website/content/whitepapers/operator.md

```diff
@@ -829,7 +827,7 @@ The controllers that own these sub-components of stacks can appear in two ways:
 
 - An operator distribution package could consist of multiple separate controllers, each handling a sub-component of the stack plus a main controller ( Responsible for the end-user facing CRD, representing the stack as a whole). Deploying such a multi-controller operator as a single package would result in all controllers running at once (one `Pod` each), but only the end-user facing API/CRD is actually exposed and documented for public consumption. When that happens, the controller responsible for this API delegates several duties to the other controllers, that are part of it's packaged using "internal" CRDs. This is useful when the whole "stack" is owned and developed by the same group of operator authors and the "subordinate" controllers don't make sense as a standalone project. To an end-user this set of controllers still appears as a single Operator. The main benefit here is separation of concerns within an operator project.
 
-![Stack-Operator](./img/08_2_umbrella.png)
+![Stack-Operator](https://raw.githubusercontent.com/cncf/tag-app-delivery/main/operator-whitepaper/v1/img/08_2_umbrella.png)
```

Comment:
Can we please use `latest` version rather than `v1` ?

Reply:
Ok, updated.